Website/Sources/Pages/HomeSections.swift

```diff
@@ -132,9 +132,17 @@ extension HomeSectionType {
         }
       }
     case .sponsor:
-      SectionHeader(type: self, language: language)
+      let sponsors = switch year {
+      case .year2025:
+        try! dataClient.fetchSponsors(.year2025)
+      case .year2026:
+        try! dataClient.fetchSponsors(.year2026)
+      }
```

Comment:
nits:
You might consider rewriting it like this:
```suggestion
      let sponsors = try! dataClient.fetchSponsors(year)
```